src/App.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react';
+// import React, { useState } from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setValue] = useState('');
+
+  const filterdMovies = <MoviesList movies={moviesFromServer} value={query} />;
```

Comment:
You don't have to define filterdMovies here
You can move the part 
```tsx
<MoviesList movies={moviesFromServer} value={query} />
```

Also check my other comment as it is related

Reply:
Done